docs/get-started/deployment-and-usage.md

```diff
@@ -327,6 +384,21 @@ The worker configuration file is watched and re-read on every change which
 provides a simple mechanism of dynamic run-time reconfiguration. See
 [worker configuration](#worker-configuration) for more details.
 
+### NFD-Topology-Updater
+
+NFD-Topology-Updater is preferably run as a Kubernetes DaemonSet. This assures
+re-examination (and CR updation) on regular intervals capturing changes in
+the allocated resources and hence the allocatable resources on a per zone
+basis. It makes sure that more CR instances are created as new nodes get
```

Comment:
Suggestion: add a reference to the *NodeResourceTopology CR* section of `introduction.md`